cmd/cluster/sandbox_file_writer.go

```diff
@@ -74,7 +74,7 @@ func (d *SandboxClusterFileStore) getSandboxFileLocation() (string, error) {
 	var dirname string
 	var err error
 	if isATest {
-		return os.TempDir() + "dotarmory_sandbox", nil
+		return os.TempDir() + "/.armory_sandbox", nil
```

Comment:
Test wasn't passing on my machine without the `/`

Reply:
i did get your branch on my local machine and executed tests - no problems with execution, so i guess this is safe for everyone